update Ayers sphene saturation functions

jannittayao opened the pull request:
new functions: Ayers_tspheneM, Ayers_tpheneTiO2_22, and Ayers_tsphene_22 to reflect updated sphene saturation equation in Ayers et al. (2022) doi: 10.1007/s00410-022-01902-z

jannittayao committed:
update Ayers sphene saturation functions

new functions: Ayers_tspheneM, Ayers_tpheneTiO2_22, and Ayers_tsphene_22 to reflect updated sphene saturation equation in Ayers et al. (2022) doi: 10.1007/s00410-022-01902-z

brenhinkeller reviewed (approved):
This looks good -- my one suggestion might be to have this one be the new default, and rename the old one `_18`?

jannittayao committed:
update ayers function names

2018 Ayers sphene saturation equations now end in _18, and the 2022 Ayers sphene saturation equations are the default ones
